tox.ini

```diff
@@ -119,12 +119,14 @@ setenv =
 basepython = python2.7
 commands =
     {[base]install_packages}
+    {[base]pip_install} certbot-apache[dev]
     python tox.cover.py
 
 [testenv:py37-cover]
 basepython = python3.7
 commands =
     {[base]install_packages}
+    {[base]pip_install} certbot-apache[dev]
```

Comment:
Why these and only these particular tests? I would have expected to see separate tox target; what did you have in mind for doing it this way?

Reply:
This is so that the tests that depend on `apacheconfig` run during the code coverage calculation